test/VariablesTest.cs

```diff
@@ -42,6 +42,21 @@ public void AddSecretTest()
             Client.Variable("foobar").AddSecret(testValue);
         }
 
+        [Test]
+        public void AddByteArraySecretTest()
```

Comment:
Please test also that byte array is clean after calling AddSecret. It's key functionality.